core/templates/dev/head/components/ExplorationCreationService.js

```diff
@@ -115,7 +115,7 @@ oppia.factory('ExplorationCreationService', [
             AlertsService.addWarning(
               parsedResponse.error || 'Error communicating with server.');
             $rootScope.loadingMessage = '';
-            $scope.$apply();
+            $rootScope.$apply();
```

Comment:
Why?

Reply:
$scope is not used in the service and since we are binding values to $rootScope, $apply should be called on rootScope.